PYME/Acquire/Spooler.py

```diff
@@ -133,14 +133,14 @@ def StartSpool(self):
 
         self.imNum = 0
 
-        # record start time here in case protocol init tasks generate events (prob only effects simulator).
-        self.tStart = time.time()
-
         self.protocol.Init(self)
 
         self._collect_start_metadata()
-   
```

Comment:
This probably wont work quite right. I think we need to set `self.tStart` before `_collect_Start_metadata` where we log it

Reply:
good catch ...